qlib/utils/__init__.py

```diff
@@ -728,3 +728,23 @@ def load_dataset(path_or_obj):
     elif extension == ".csv":
         return pd.read_csv(path_or_obj, parse_dates=True, index_col=[0, 1])
     raise ValueError(f"unsupported file type `{extension}`")
+
+
+#################### Operator Wrapper #####################
+
+
+class OpsWrapper(object):
```

Comment:
I think this class could be moved into `ops.py`
It will not be used by any other module